build/azDevOps/azuredevops-vars.yml

```diff
@@ -14,6 +14,13 @@ variables:
   - name: pool_vm_image
     value: ubuntu-20.04
 
+  # Determine the "stability" of the image
+  - name: stability
+    ${{ if eq(parameters.Build.SourceBranchName, 'main') }}:
```

Comment:
for consistency this should include and or() condition for master, I think? No direct application to this repo but if this gets lifted and shifted elsewhere?

Reply:
Good point, will add that in